extensions-core/druid-ranger-security/pom.xml

```diff
@@ -160,6 +160,10 @@
                     <groupId>org.elasticsearch.plugin</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
can we add a comment to explain why we are excluding this dependency and why it is safe to do so?

Reply:
This is a transitive dependency of RangerRESTClient. 
https://github.com/apache/ranger/blob/master/agents-common/src/main/java/org/apache/ranger/plugin/util/RangerRESTClient.java
to handle list unmarshalling see:
https://github.com/apache/ranger/blob/1d091170344ec622d2f34e3bdeb292c48ecdd7c5/agents-common/src/main/java/org/apache/ranger/plugin/util/RangerRESTClient.java#L226

To my best understanding, this dependency is needed for Ranger Client, not  Ranger Authorizer ( code in Druid). 


The static analysis using maven dependency:analyse shows the dependency is not used directly (when explicitly added as a dependency) 
`[INFO] --- dependency:3.6.1:analyze (default-cli) @ druid-ranger-security ---
[WARNING] Unused declared dependencies found:
[WARNING]    com.google.code.findbugs:jsr305:jar:2.0.1:provided
[WARNING]    com.fasterxml.jackson.dataformat:jackson-dataformat-smile:jar:2.12.7:provided
[WARNING]    org.apache.druid:druid-services:jar:29.0.0-SNAPSHOT:provided
[WARNING]    org.apache.druid:druid-server:test-jar:tests:29.0.0-SNAPSHOT:test
[WARNING]    io.netty:netty:jar:3.10.6.Final:provided
[WARNING]    javax.servlet:javax.servlet-api:jar:3.1.0:provided
[WARNING]    com.fasterxml.jackson.core:jackson-core:jar:2.12.7:provided
[WARNING]    org.codehaus.jackson:jackson-jaxrs:jar:1.9.13:compile
[WARNING]    com.sun.jersey:jersey-server:jar:1.19.4:provided
[WARNING]    org.jdbi:jdbi:jar:2.63.1:provided
[WARNING]    org.apache.commons:commons-lang3:jar:3.12.0:provided
[WARNING]    org.apache.hadoop:hadoop-client-runtime:jar:3.3.6:runtime
[WARNING]    joda-time:joda-time:jar:2.12.5:provided
[WARNING]    javax.ws.rs:jsr311-api:jar:1.1.1:provided
[WARNING] Non-test scoped test only dependencies found:
[WARNING]    com.google.code.gson:gson:jar:2.10.1:compile`

Reply:
ok, let's add a short comment to note to summarize that.

Reply:
done